web/styles/popovers.css

```diff
@@ -337,24 +337,42 @@ ul.popover-group-menu-member-list {
     flex-direction: column;
 
     .simplebar-content {
+        display: grid;
+        grid-template-columns:
+            var(--user-group-popover-horizontal-padding)
+            [group-member-icon-start] minmax(5px, auto)
+            [group-member-icon-end] var(--user-group-popover-icon-text-gap)
+            [group-member-name-start] 1fr [group-member-name-end] var(
```

Comment:
`1fr` should be expressed as `minmax(0, 1fr)` to prevent grid blowouts.

While the inline labels are OK here, can you switch this up to use named areas? I generally only use this style of labeling when something needs to occupy a different set of rows/columns from the basic grid.

Reply:
It might also be worth thinking about whether that padding at the beginning and end belongs as part of the grid, or should sit outside of it. I don't have an opinion, but it seems worth asking (null areas like that can be named with `.`, should you decide they remain)

Reply:
Switched to `grid-template-areas` and `grid-template-columns`:
```css
grid-template-areas: "group-member-icon gap group-member-name";
grid-template-columns: max-content var(--user-group-popover-icon-text-gap) minmax(0, 1fr);
```

Reply:
I also moved the padding to the `popover-group-menu-member` element itself, as your note about the padding made sense. Defining the padding as null areas in the grid didn't add any advantage to us, so it doesn't make sense to define the same, which we would do via a simple padding to the element itself.